docs/theming.md

```diff
@@ -109,15 +109,91 @@ dark theme.
 In order to style your own components with our tooling, the component's styles must be defined 
 with Sass. 
 
-You can consume the theming functions and variables from the `@angular/material/core/theming`.
```

Comment:
@jelbourn  Ok, I'll do this! But should I use your tiny as possible example or can we provide the example with all that information to theming custom form components?